python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -466,6 +511,13 @@ def on_renew_sso_session_timeout(self):
     #
     ############################################################################
 
+    def _polyfill(self):
+        """
+        Called by Qt when the Web Page has changed and before it is loaded.
```

Comment:
explain what the method is doing, not when it's being called :)

Reply:
Comment updated